addons/website/views/res_config_settings_views.xml

```diff
@@ -123,6 +123,10 @@
                                                href="https://console.developers.google.com/flows/enableapi?apiid=maps_backend,static_maps_backend&amp;keyType=CLIENT_SIDE&amp;reusekey=true">
                                                 Create a Google Project and Get a Key
                                             </a>
+                                            <a class="btn-link fa fa-arrow-right" target="_blank"
+                                               href="https://cloud.google.com/maps-platform/pricing">
+                                                Enable billing on your Google Project
+                                            </a>
```

Comment:
How did you test? Google does not require to enter credit card information or something like that?

Reply:
I have found no way without credit card, so I used credit card.

Reply:
And you will not be charged?

Reply:
If I use less than $200 a month I should not be charged:

 https://cloud.google.com/maps-platform/pricing

In the billing report I have $0 but it should be $0.022 so I'm not sure how it's updated.

Reply:
Ok thanks, I'll try that then, thanks :+1: